### PR TITLE
Okcoin: set margin to false

### PR DESCRIPTION
**Pull request by Dan-krm**
Set margin to false:
image

### CODE REVIEW

**Comment by carlosmiei** [comment]
@Dan-krm If we want to "unlist" margin completely, we should revisit all places that might be using it, like fetchMarkets. Can you please do that?
image

**Comment by Dan-krm** [comment]
> @Dan-krm If we want to "unlist" margin completely, we should revisit all places that might be using it, like fetchMarkets. Can you please do that?

Yeah I can do that

File: js/okcoin.js [hunk 1]
```diff
@@ -199,6 +199,7 @@ module.exports = class okcoin extends Exchange {
                     },
                 },
                 'margin': {
+                    // Margin trading closed down on February 21, 2022
                     'get': {
```
Comment (carlosmiei) [inline js/okcoin.js:203]: why are we keeping margin endpoints? are they still valid?

Comment (Dan-krm) [reply js/okcoin.js:203]: > why are we keeping margin endpoints? are they still valid?

I think some of them still work for people who placed margin orders before it was shut down and I think there's a chance they could bring margin trading back, from some of their blog posts it looks like in the past they've closed down margin trading and brought it back. We can delete them if you think that's necessary though

Comment (carlosmiei) [reply js/okcoin.js:203]: @Dan-krm agreed let's keep them for now

**Comment by carlosmiei** [comment]
all good ✅

**Review by carlosmiei — APPROVED** [review]

**Pull request merged**